### DISCUSSION TITLE
A question to action versions

### DISCUSSION THREAD

**Discussion by josephjthomas** [community/community#12560] [Actions]
If I use the [cache action](https://github.com/actions/cache) in my workflow and specify version v2, do I need to update to v2.1.7 to be able to have `Support 10GB cache upload using the latest version 1.0.8 of @actions/cache` available? 
https://github.com/actions/cache/releases
Or can I stick to version v2 and still have all newer releases included?

**Comment by ethomson** [comment]
Hi @josephjthomas - you need to use v2.1.7 (or newer) of `actions/cache` in order to get the 10GB cache limit. *But* if you're referencing your actions as `uses: actions/cache@v2` then that means that you're asking for the latest version of `actions/cache` in the v2 major version lineage. So you should get v2.1.7 just by referencing `v2`. Let me know if that's now what you're seeing!
